CODE_OF_CONDUCT.md

```diff
@@ -61,7 +61,7 @@ representative at an online or offline event.
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the maintainers responsible for enforcement at:
 
-alexvonme at matrix dot org
+at alexvonme : matrix dot org
```

Comment:
Starting your account name with an at, is the quasi-universal way to state that your account is within a specific network such as twitter or mastodon. Everything else is redundant.

I corrected format specific to matrix.